_posts/2023-01-09-knn-similarity-refresh.markdown

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title:  "Build k-Nearest Neighbor (k-NN) Similarity Search Engine with OpenSearch"
+authors:
+- vamshin
+- laijiang
+- csw
+- jmazane
+date: 2023-01-09 00:00:00 -0700
+categories:
+ - technical-post
+meta_keywords: "approximate k-NN search, k-nearest neighbor plugin, k-NN plugin, ANN similarity search solution"
+meta_description: "Learn how the OpenSearch approximate k-NN search solution enables you to build a scalable, reliable, and distributed framework for similarity searches" 
+
+excerpt: Since the OpenSearch Project introduced the k-nearest neighbor plugin in 2019, it has supported both exact and approximate k-NN search. The approximate k-NN search method is more efficient for large datasets with high dimensionality because it reduces the cardinality of searchable vectors. This approach is superior in speed at the cost of a slight reduction in accuracy. 
+---
+
+# Build k-Nearest Neighbor (k-NN) Similarity Search Engine with OpenSearch
```

Comment:
Please remove this title  - it appears as duplicative

Reply:
Done